### PR TITLE
ansible: update to 3.2.0.

### PR DESCRIPTION
**Pull request by jcgruenhage**

### CODE REVIEW

**Comment by ericonr** [comment]
Could you take a look at the build failures?

**Comment by jcgruenhage** [comment]
@ericonr I have looked, but I'm not sure what's happening there, especially considering I built it locally on x86_64, which built, installed and worked just fine..

**Comment by ericonr** [comment]
It seems `ansible-base` requires `ansible` for testing, which isn't doable. So remove `checkdepends` from it and add something like

```
do_check() {
: # tests require ansible, which would be a cyclic dependency
}
```

**Comment by jcgruenhage** [comment]
that's not it, it seems...

**Comment by ericonr** [comment]
Please do look at the CI failures... It was failing on ansible-base, it's now failing in ansible. You can run tests locally by building with `./xbps-src -Q pkg ...`

File: srcpkgs/ansible-base/template [hunk 1]
```diff
@@ -22,3 +21,7 @@ post_install() {
 		vman ${m}
 	done
 }
+
+do_check() {
+: # tests require ansible, which would be a cyclic dependency
```
Comment (ericonr) [inline srcpkgs/ansible-base/template:26]: Indent this with a tab, please.

**Comment by jcgruenhage** [comment]
> Please do look at the CI failures... It was failing on ansible-base, it's now failing in ansible. You can run tests locally by building with `./xbps-src -Q pkg ...`

I assumed this tests would be running locally either way, sorry.

I looked at the CI failures, they looked the same to me as last time, so I assumed this didn't help at all. Sorry.

**Comment by ericonr** [comment]
No problem, the errors do look similar.

It's been suggested that you export `PYTHONPATH=lib` before running tests. We are trying to come up with a more general solution, possibly.

File: srcpkgs/ansible-base/template [hunk 2]
```diff
@@ -22,3 +21,7 @@ post_install() {
...
+}
```
Comment (ericonr) [inline srcpkgs/ansible-base/template:27]: Actually, replace with `make_check=no` and a comment on top.

File: srcpkgs/ansible/template [hunk 3]
```diff
@@ -1,14 +1,14 @@
 # Template file for 'ansible'
 pkgname=ansible
-version=2.10.1
+version=3.3.0
```
Comment (ericonr) [inline srcpkgs/ansible/template:3]: Is this official? I can't find it anywhere but pypi.

Comment (jcgruenhage) [reply srcpkgs/ansible/template:3]: yep, this is official. See https://docs.ansible.com/ansible/latest/roadmap/COLLECTIONS_3_0.html and https://www.ansible.com/blog/ansible-3.0.0-qa. They split out all the modules, plugins etc into "collections", and renamed the rest to ansible-base. The new ansible package is a package where they bundle those collections,  whose content was included in the old ansible. While those two packages both started at 2.10, the version numbers are now diverging. Ansible 3.0 is based on ansible-base 2.10, Ansible 4.0 will be based on ansible-base 2.11, and so on.

**Pull request merged**